packages/react-components/react-drawer/library/src/index.ts

```diff
@@ -70,3 +70,5 @@ export {
   useDrawerFooter_unstable,
 } from './DrawerFooter';
 export type { DrawerFooterProps, DrawerFooterSlots, DrawerFooterState } from './DrawerFooter';
+
+export type { DrawerBaseState } from './shared';
```

Comment:
@marcosmoura is it something that should be exported?

Reply:
Even though there's no apparent harm, I believe this shouldn't be exported. It serves as the base for DrawerInline and DrawerOverlay to share the same props. We can make that @internal and not export it.